crawl-ref/source/art-data.txt

```diff
@@ -1712,6 +1712,18 @@ MP:      +5
 INSCRIP: Invo=14 Evo=0
 DBRAND:  It sets your invocations skill to 14 and your evocations skill to 0.
 
+ENUM:    GOLDTOUCH_RING
+NAME:    ring of Gozag's Touch
+INSCRIP: *GoldTouch
+OBJ:     OBJ_JEWELLERY/RING_PROTECTION
```

Comment:
In the screenshot posted on Reddit: https://www.reddit.com/r/dcss/comments/y8b0zy/new_pr_ring_of_gozags_touch_unrand_artefact/ you can see at the bottom, the {AC+0} search term, a vestigial leftover from being implemented as a ring of protection. 
There is an "amulet of nothing" used for the base item of a couple amulet unrands (like the dreamshard necklace), I wonder if a ring of nothing could be useful to implement also for this particular case.
Alternately, maybe you could find an existing ring base type that works well with the theme.
Alternately alternately, maybe it's fine as it is.

Reply:
This seems okay as is. I did consider a ring of nothing, but the ring of protection works fine enough to get this PR into trunk.